src/lib/tabs/tab-group.ts

```diff
@@ -275,4 +279,41 @@ export class MdTabGroup extends _MdTabGroupMixinBase implements AfterContentInit
     this._tabBodyWrapperHeight = this._tabBodyWrapper.nativeElement.clientHeight;
     this._renderer.setStyle(this._tabBodyWrapper.nativeElement, 'height', '');
   }
+
+  /** Body content was swiped left/right */
+  _bodyContentSwiped(event: HammerInput): void {
+    if (this.selectedIndex === null) {
+      this.selectedIndex = 0;
+    }
+
+    if (event.direction === 2 || event.direction === 4) {
```

Comment:
Add an enum or constants instead of using the number literals in the function body